js/crex24.js

```diff
@@ -809,14 +809,14 @@ module.exports = class crex24 extends Exchange {
         await this.loadMarkets ();
         const request = {}
         if(since){
-            request.from = this.iso8601 (since).replace(/\.\d{3}/,'') // We need to trim out ms beacuse exchange does not support
+            request.from = this.ymdhms (since, 'T');
```

Comment:
Hey! First of, thx for your participation! A quick question: it looks like your crex24.js file is a bit outdated, if compared to the most recent version in CCXT:

- https://github.com/ccxt/ccxt/blob/master/js/crex24.js#L896

So, it appears that CCXT already does support this. Looking forward to your confirmation )

Reply:
That commit is not something important. I implemented fetchOrders. And this commit just a revision to my previous commit.

Reply:
@alialparslan sorry, but this PR violates the CONTRIBUTING guidelines:

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes
- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#accessing-dictionary-keys

Therefore the build fails:

- https://travis-ci.org/ccxt/ccxt/builds/595624670#L1481

And this functionality already seems to be in the master version of CCXT.

Reply:
Sorry for wasting your time. I have fixed transpiler errors. 
I am looking master branch from browser and clearly there isn't fetchOrders method for crex24 and "fetchOrders" property of has object have false value.

Reply:
Ah, looks like you're right, pardon! However, `fetchOrders` has to return both open and closed orders – can you please help verify that they do return open orders from that endpoint? Thanks!

Reply:
Yes, it returns all orders including open ones.

Reply:
@alialparslan awesome, thx!